netbox/utilities/constants.py

```diff
@@ -27,3 +27,17 @@
     ('111111', 'Black'),
     ('ffffff', 'White'),
 )
+
+# Keys for PostgreSQL advisory locks. These are arbitrary bigints used by
+# the pg_advisory_lock contextmanager/decorator. When a lock is acquired,
+# one of these keys will be used to identify said lock.
+#
+# When adding a new key, pick something arbitrary and unique so
+# that it is easily searchable in query logs.
+ADVISORY_LOCK_KEYS = {
+    'available-prefixes': 123456710,
+    'available-ips': 234567820,
+}
+
+MAX_INT64 = 2**63 - 1
```

Comment:
We probably don't need to define constants for these; should be find to reference the absolute values directly within the function.